data/en/cfqueryparam.json

```diff
@@ -37,39 +38,18 @@
         "cf_sql_longnvarchar",
         "cf_sql_money",
         "cf_sql_money4",
+        "cf_sql_nchar",
+        "cf_sql_nvarchar",
         "cf_sql_numeric",
         "cf_sql_real",
         "cf_sql_refcursor",
         "cf_sql_smallint",
+        "cf_sql_sqlxml",
         "cf_sql_time",
         "cf_sql_timestamp",
         "cf_sql_tinyint",
         "cf_sql_varchar",
-        "cf_sql_nvarchar",
-        "bigint",
```

Comment:
It's not a mistake, that they are listed both with "cf_sql_" prefix and without because they work both. But yeah, on the other hand, this is explained in the param and the engine notes. So it's not completely wrong that we don't need both&hellip;

Reply:
I think we want to keep both in there... because editors like Sublime or VSCode will use this info for autocomplete. Can you add them back in and then I'll merge the PR - thanks!

Reply:
Ah, because of that we left them in.

Reply:
Yeah also for that reason we don't want to add version support tags in there.